webhook/main.go

```diff
@@ -49,20 +63,29 @@ func main() {
 	}
 
 	// Create a new Cmd to provide shared dependencies and start components
-	mgr, err := manager.New(cfg, manager.Options{
+	mgr, err := ctrl.NewManager(cfg, ctrl.Options{
 		Namespace: namespace,
+		Scheme:    scheme,
+		CertDir:   server.WebhookServerCertDir,
 	})
 	if err != nil {
 		log.Error(err, "Failed to create manager")
 		os.Exit(1)
 	}
 
-	err = createWebhooks(mgr, cfg)
+	err = createWebhooks(mgr)
 	if err != nil {
 		log.Error(err, "Failed to create webhooks")
 		os.Exit(1)
 	}
 
+	if err := ctrl.NewWebhookManagedBy(mgr).For(&workspacev1alpha1.DevWorkspace{}).Complete(); err != nil {
+		log.Error(err, "failed creating conversion webhook")
```

Comment:
Just curious, if the conversion webhooks fail to create should we exit or just continue without them?

Reply:
I believe if the webhooks fail to set up you won't be able to create DevWorkspaces -- when you create a DevWorkspace it calls the conversion webhooks (at least for the v1alpha1 case) and if it can't get a response it blocks creation.

Also surprising info about how conversion works: If you specify `conversion: none` in the CRD, you can "get" a v1alpha2-spec workspace that is just the `v1alpha1` and a wrong GVK :shrug: